storage/innobase/xtrabackup/test/suites/xbcloud/basic_operations.sh

```diff
@@ -67,7 +67,7 @@ xbstream -xv -C $topdir/partial < $topdir/partial/partial.xbs \
 
 sort -o $topdir/partial/partial.list $topdir/partial/partial.list
 
-diff -u $topdir/partial/partial.list - <<EOF
+run_cmd diff -u $topdir/partial/partial.list - <<EOF
```

Comment:
I think because of set -e mode, anything that returns nonzero should cause script exit. Can you please check why this doesn't happen?

Reply:
That is correct, currently in the test framework we do not exit with error in case of any command return non-zero.
xtrabackup is a function that calls run_cmd xtrabackup. So this test was wrongly assuming diff will fail the test in case of any diff, while it should be doing it via run_cmd.

Reply:
set -e is broken in pxb testframework. So everything has to be run_cmd :(